cupy_setup_build.py

```diff
@@ -526,6 +527,14 @@ def preconfigure_modules(compiler, settings):
             cmd = 'nvcc' if not use_hip else 'hipcc'
             errmsg = ['{} command could not be found in PATH.'.format(cmd),
                       'Check your PATH environment variable.']
+        elif ('required_cuda_version' in module
```

Comment:
This needs to be checked first (before `not check_library()` condition) to skip building cuTENSOR/cuSPARSELt support for unsupported CUDA versions.